modules/private-shared-lib/src/domain/individual_transfer_entity.ts

```diff
@@ -53,18 +54,32 @@ export enum IndividualTransferInternalState {
     TRANSFERS_SUCCESS = 'TRANSFERS_SUCCESS',
 }
 
+export interface IndividualQuoteResponse extends SDKSchemeAdapter.V2_0_0.Outbound.Types.individualQuoteResult {
+    expiration: SDKSchemeAdapter.V2_0_0.Outbound.Types.DateTime;
```

Comment:
I believe this missing quote a few fields?

e.g. ilpCondition, etc?

Reply:
well, this is what we get from the switch:
https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L1862

https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L3500

https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L3473